.circleci/config.yml

```diff
@@ -0,0 +1,216 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: edxops/xenial-common:hawthorn.beta1
```

Comment:
Curious here, do we need to use this image? or can we use one of the defaults ones from CircleCI (they have some that come already with firefox installed). I wonder if we could avoid all those configuration steps by changing the image.

See: https://circleci.com/docs/2.0/circleci-images/#language-image-variants

Reply:
This image is based on the standard Ubuntu Xenial image: https://github.com/edx/configuration/blob/master/docker/build/xenial-common/Dockerfile

I tried a few different images including the Circle ones but they had more issues. The standard Ubuntu Xenial should work (since we are not using the Ansible stuff anyway) but this image already has most of the deb updates from the last two years (Xenial is from April 2016). Plus to avoid false negatives in tests it makes sense to install Node and Firefox versions that are used by upstream (these versions are what Ginkgo releases have). Circle images use different ones so we will have to set them up anyway.

For future, I think it would be simpler to contribute some Docker files which have all of this stuff installed for testing to edx/configuration so there are standardized images for all forks to use.